src/rgw/rgw_sal_motr.cc

```diff
@@ -3351,6 +3356,10 @@ int MotrAtomicWriter::complete(size_t accounted_size, const std::string& etag,
   // other attrs.
   obj.get_key().get_index_key(&ent.key);
   ent.meta.size = total_data_size;
+  uint64_t lid = M0_OBJ_LAYOUT_ID(obj.meta.layout_id);
+  uint64_t unit_sz = m0_obj_layout_id_to_unit_size(lid);
+  uint64_t size_rounded = roundup(ent.meta.size, unit_sz);
```

Comment:
Can we create a function which accepts `size` as input and returns the `rounded_size`?